homework3/task2.py

```diff
@@ -0,0 +1,16 @@
+# Напишите программу, которая найдёт произведение пар чисел списка. Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+
+#  Пример:
+# [2, 3, 4, 5, 6] => [12, 15, 16];
+# [2, 3, 5, 6] => [12, 15]
+
+
+list1 = [2, 3, 5, 4, 38, 10, 6]
+if len(list1) % 2 == 0:
+    list2 = [0] * int(len(list1) // 2)
+else:
+    list2 = [0] * (len(list1) // 2 + 1)
```

Comment:
```suggestion
list2 = []
```
Вот эти 4 строчки не нужны. Да и зачем создавать второй список таким образом? Это же вам не С#. Смотрите ниже.